index.html

```diff
@@ -1,21 +1,659 @@
+    </li>
+    <li class="nav-item border-right">
+      <a class="nav-link text-body" href="#">BUSINESS</a>
+    </li>
+    <li class="nav-item border-right">
+      <a class="nav-link text-body" href="#">TECH n SCIENCE</a>
+    </li>
+    <li class="nav-item border-right">
+      <a class="nav-link text-body" href="#">CULTURE</a>
+    </li>
+    <li class="nav-item border-right">
+      <a class="nav-link text-body" href="#">SPORT</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link text-body" href="#">HEALTH</a>
+    </li>
+  </ul>
+
+  <main class="container-fluid mt-3">
+    <div class="row">
+      <div class="col-md-12 col-lg-8">
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-md-5 ">
+              <div class="featured-stories ">
+                <h5 class="text-left text-secondary h6 font-weight-bold">FEATURED STORIES</h5>
+                <div class=" border-0">
+                  <img src="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/03/13/gettyimages-831982426.jpg"
+                    class="card-img-top" alt="...">
+                  <div class="card-body">
+                    <h5 class="card-title">GOP Senators Roll Out Bill to Curb Emergency Powers</h5>
+                    <p class="card-text">"Congress gave these legislative powers away in 1976, and it is far past
+                      time that
+                      we as an
+                      institution took them back," said Senator Mike Lee, a Utah Republican.</p>
+                    <hr>
+                    <h6>Trump’s Climate Denial Gets More Ridiculous By The Day</h6>
+                    <hr>
+                  </div>
+                </div>
+              </div>
+
+              <div class="featured-stories ">
+                <div class="card mb-0 border-0">
+                  <img src="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/03/13/ivanka-trump-and-senior-advisor-jared-kushner.jpg"
+                    class="card-img-top" alt="...">
+                  <div class="card-body">
+                    <h5 class="card-title">GOP Senators Roll Out Bill to Curb Emergency Powers</h5>
+                    <p class="card-text">"Congress gave these legislative powers away in 1976, and it is far past time
+                      that
+                      we as an
+                      institution took them back," said Senator Mike Lee, a Utah Republican.</p>
+                    <hr>
+                    <h6>Trump’s Climate Denial Gets More Ridiculous By The Day</h6>
+                    <hr>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="col-xs-12 col-md-7">
+              <div class="container-fluid">
+                <div class="row">
+                  <h5 class="text-left text-secondary h6 font-weight-bold">TOP STORIES</h5>
```

Comment:
use `order-first order-1-lg` here in order to have top stories as a first ,and second where needed.
Follow this example in other cases please.

Reply:
thanks! we didn't notice the reordering :sweat_smile: